src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java

```diff
@@ -58,10 +59,15 @@ public MoveCollectedGamepieceToArmCommandGroup(
                 () -> UnifiedArmSubsystem.KeyArmPosition.PrepareToAcquireFromCollector,
                 () -> UnifiedArmSubsystem.RobotFacing.Forward);
 
-        var waitThenPullArm = new WaitCommand(0.1).andThen(cubePullOutGamePieceCommand);
+        //Instead of waitThenPullArm, will use the clawConeIntake with a small change
+        //var waitThenPullArm = new WaitCommand(0.1).andThen(cubePullOutGamePieceCommand);
+        var clawCubeIntake = clawMotors.createIntakeCommand()
```

Comment:
you create this command, but then never try to run it in the command group so it won't happen. Look at how the cone version is using it